.distignore

```diff
@@ -0,0 +1,30 @@
+/.git
+/.github
+/node_modules
+/vendor
+
+/bin
+/docs
+/tests
+
+/*.DS_store
+/.DS_store?
+
+/.editorconfig
+/.eslintrc.js
+/.nvmrc
+/.wp-env.json
+/composer.json
+/composer.lock
+/package.json
+/package-lock.json
+/phpcs.xml.dist
+/phpunit.xml.dist
+
+/CODE_OF_CONDUCT.md
+/CONTRIBUTING.md
+/SECURITY.md
+/LICENSE
```

Comment:
I think we need to remove this file from here since every wordpress.org plugin needs to include a license file.